packages/headless/src/api/knowledge/stream-answer-api.ts

```diff
@@ -63,55 +58,10 @@ type PayloadType =
   | 'genqa.citationsType'
   | 'genqa.endOfStreamType';
 
-const headerMessageSchema = new Schema<HeaderMessage>({
-  answerStyle: new StringValue(),
-  contentFormat: new StringValue(),
-});
-
-const messageSchema = new Schema({
-  textDelta: new StringValue(),
-});
-
-const citationsSchema = new Schema({
-  citation: new ArrayValue(
-    new RecordValue({
-      values: {
-        clickUri: new StringValue(),
-        id: new StringValue(),
-        permanentid: new StringValue(),
-        text: new StringValue(),
-        title: new StringValue(),
-        uri: new StringValue(),
-      },
-    })
-  ),
-});
-
-const validateHeaderMessage = (headerMessage: HeaderMessage) => {
-  headerMessageSchema.validate(headerMessage);
-};
-
-const validateMessage = (message: {textDelta: string}) => {
-  messageSchema.validate(message);
-};
-
-const validateCitationsMessage = (citations: {
-  citation: GeneratedAnswerCitation[];
-}) => {
-  citationsSchema.validate(citations);
-};
-
-const validateEndOfStream = (endOfStream: {answerGenerated: boolean}) => {
-  new Schema({
-    answerGenerated: new BooleanValue(),
-  }).validate(endOfStream);
-};
-
```

Comment:
Instead of using shemas to type check function that arent client facing, I used simpler conditions